### DISCUSSION TITLE
Changing the pirate ship design

### DISCUSSION THREAD

**Discussion by redmoogle** [yogstation13/Yogstation#10231] [Discussions]
As it stands, _in my opinion_ the current pirate shuttle(see below) is a little boring, however i did want to change it to another pirate ship we have(also below) although changed around to fit the pirate ship armory and all that

![image](https://user-images.githubusercontent.com/20369082/97503934-589fab00-194c-11eb-9689-ec9464491285.png)

![image](https://user-images.githubusercontent.com/20369082/97504069-969ccf00-194c-11eb-8970-ac8b2106a19f.png)

Opinions?

**Comment by TheLinuxOS** [comment]
Seems like a good idea to me and should be a fairly simple mapping change, the current pirate ship has always felt a little cramped to me and this change should alleviate that.

**Reply by BigBigga** [reply]
Idunno, don't think I like it, I think the (OG) pirate ship looking cramped and rundown is kind of the point. Since, they're pirates and all, they're not exactly swimming in money to begin with, and it makes sense for their stuff to feel and look shitty/makeshift, .50 turrets on every corner is some downright nukie shizz,

**Reply by redmoogle** [reply]
> 
> 
> Idunno, don't think I like it, I think the (OG) pirate ship looking cramped and rundown is kind of the point. Since, they're pirates and all, they're not exactly swimming in money to begin with, and it makes sense for their stuff to feel and look shitty/makeshift, .50 turrets on every corner is some downright nukie shizz,

They're regular bullets. Basically the laser turret but bullets
